<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="widgetcentral">
   <property name="styleSheet">
    <string notr="true">font-family:&quot;Inter&quot;, sans-serif;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Background_Login">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_fondo">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">border-image: url(:/Imagenes/Fondo MAPA.jpeg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_registrar">
         <property name="styleSheet">
          <string notr="true">font-family:&quot;Inter&quot;, sans-serif;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>60</y>
               <width>231</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Inicia sesi칩n con tu usuario de </string>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>281</width>
               <height>41</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight:500;
font-size:35px;
</string>
             </property>
             <property name="text">
              <string>Iniciar Sesi칩n</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>110</y>
               <width>111</width>
               <height>25</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight:500;
font-size:22px;</string>
             </property>
             <property name="text">
              <string>Usuario:</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="line_Usuario">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>110</y>
               <width>391</width>
               <height>40</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 3px solid rgba(246, 143, 137, 255);
	border-radius:	10%;
	text-align: left;
	font-weight:100;
}</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="placeholderText">
              <string>Ingrese su nombre de usuario</string>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::LogicalMoveStyle</enum>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>160</y>
               <width>151</width>
               <height>61</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight:500;
font-size:22px;</string>
             </property>
             <property name="text">
              <string>Contrase침a:</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="line_Clave">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>170</y>
               <width>391</width>
               <height>40</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 3px solid rgba(246, 143, 137, 255);
	border-radius:10%;
	font-weight:100;
}</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="placeholderText">
              <string>Ingrese su contrase침a</string>
             </property>
            </widget>
            <widget class="QPushButton" name="bt_ingresar">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>230</y>
               <width>391</width>
               <height>40</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
		background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0.0170455, stop:0.261364 rgba(246, 143, 137, 			255), stop:1 rgba(128, 74, 70, 255));
		color: rgb(255, 255, 255);
		border-radius:10%;

}

QPushButton:hover{
		background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0.0170455, stop:0.261364 rgba(200, 90, 90, 			255), stop:1 rgba(155, 90, 100, 255));
}
</string>
             </property>
             <property name="text">
              <string>Ingresar</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>250</x>
               <y>60</y>
               <width>131</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight:500;</string>
             </property>
             <property name="text">
              <string>MAPA Interface</string>
             </property>
            </widget>
            <widget class="QPushButton" name="bt_crear_usuario">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>280</y>
               <width>111</width>
               <height>25</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">
border: 0px;

QPushButton#pushButton:hover {
	color: rgb(53, 132, 228);
}</string>
             </property>
             <property name="text">
              <string>Crear usuario</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Recursos/Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
